learn/what_is_meilisearch/comparison_to_alternatives.md

```diff
@@ -95,7 +95,7 @@ Can't find a client you'd like us to support? [Submit your idea or vote for it](
 |   | Meilisearch | Algolia | Typesense | Elasticsearch |
 |---|:---:|:----:|:---:|:---:|
 | Placeholder search | ✅ | ✅ | ✅ | ✅ |
-| Multi-index search | [2023](https://github.com/meilisearch/product/issues/74) | ✅ | ✅ | ✅ |
+| Multi-index search | [In beta](https://github.com/meilisearch/instant-meilisearch/releases/tag/v0.10.1-multi-index-search.0) | ✅ | ✅ | ✅ |
```

Comment:
This line is the only part of this PR that I'm concerned about.

Communicating this Instant Meilisearch feature as beta multi-index search could be viewed as deceptive. Unless I'm mistaken, Meilisearch itself (engine/api) still does not support multi-index search. I don't know exactly what Instant Meilisearch is doing under the hood, but I'm guessing it's not "true" multi-index search, more likely duplicating the search query in an imitation of multi-index search that has some performance costs.

@bidoubiwa @irevoire What do you think about this?

Reply:
I agree with @dichotommy.

When users are using Meilisearch through instantsearch.js, they are now able to use the `Index` widget. Which leads to enabling multi-index search in this scenario.

But nor Meilisearch nor the other SDK's provide that feature

Reply:
Thanks for the clarification @dichotommy @bidoubiwa I will update this accordingly